src/test/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/resource/SyncAndMigrateResourceIntTest.kt

```diff
@@ -132,6 +137,47 @@ class SyncAndMigrateResourceIntTest : SqsIntegrationTestBase() {
     repository.save(wingB)
   }
 
+  @Nested
```

Comment:
This is an integration test not a service test, please move to an new test class

Reply:
Also you could write an integration test (calling API) to cover these off too